mmv1/templates/terraform/examples/transcoder_job_template_pubsub.tf.erb

```diff
@@ -0,0 +1,94 @@
+resource "google_pubsub_topic" "transcoder_notifications" {
```

Comment:
I believe we'll need to mark every single resource in here with `provider = beta` in order to resolve the error in VCR:

```hcl
=== CONT  TestAccTranscoderJob_transcoderJobEncryptionsExample
    vcr_utils.go:152: Step 1/2 error: Error running pre-apply refresh: exit status 1
        
        Error: Inconsistent dependency lock file
        
        The following dependency selections recorded in the lock file are
        inconsistent with the current configuration:
          - provider registry.terraform.io/hashicorp/google: required by this configuration but no version is selected
        
        To make the initial dependency selections that will initialize the dependency
        lock file, run:
          terraform init
```